docs/astro/sql/operators/transform.rst

```diff
@@ -0,0 +1,45 @@
+==================
+transform operator
+==================
+
+.. _transform_operator:
+
+When to use the ``transform`` operator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The transform function of the SQL decorator is the ``T`` of the ELT system. Each step of the transform pipeline creates a new table from the ``SELECT`` statement and enables tasks to pass those tables as if they were native Python objects. The transform: applies a SQL select statement to a source table and saves the result to a destination table.
+
+
+There are two use cases of the ``transform`` operator.
+
+#. When we pass tables between tasks while completing a data transformation.
+#. When we pass a Pandas dataframe while completing a data transformation.
```

Comment:
```suggestion
```

Reply:
Maybe we can add this description as part of the heading `Case 1: When we pass tables between tasks while completing a data transformation` and `Case 2: When we pass a Pandas dataframe while completing a data transformation.`

Reply:
Done